daemon/daemon_test.go

```diff
@@ -98,6 +98,10 @@ func (ds *DaemonSuite) SetUpTest(c *C) {
 	option.Config.Opts.Set(option.DropNotify, true)
 	option.Config.Opts.Set(option.TraceNotify, true)
 
+	// Disable restore of host IPs for unit tests. There can be arbitrary
+	// state left on disk.
+	option.Config.DisableHostIPRestore = true
```

Comment:
Do you anticipate using this anywhere else in the future? It might be useful as a CLI parameter, perhaps? (basically a clobber the existing routerIP please... although it might be simpler to gate it via the top-level `restore` option).
If not, it seems simpler to have the test clean up after itself instead.

Reply:
The unit test should not delete any system devices. I think i's correct to not have the unit test derive values from the system itself.

We can always add an option. Right now, if there is a conflict, an entire state removal is needed (running the state clean init container)